lib/showoff.rb

```diff
@@ -466,6 +466,9 @@ def process_content_for_replacements(content)
         result.gsub!(match[0], "<pre class=\"highlight\"><code class=\"#{css}\">#{file}</code></pre>")
       end
 
+      result.gsub!(/\[(.*fa-.*)\]/, '<i class="fa \1"></i>')
```

Comment:
Can we drop the initial `.*` from this, so that it only matches `[fa-` instead of `[ anything fa-`